backend/app/cache.js

```diff
@@ -13,35 +15,30 @@ const MILLISECONDS_IN_DAY = MILLISECONDS_IN_HOUR * HOURS_IN_DAY;
 const MILLISECONDS_IN_WEEK = MILLISECONDS_IN_DAY * DAYS_IN_WEEK;
 
 const log = getLogger();
-const cache = {};
+const REDIS_URL = process.env.REDIS_URL || null;
+const client = redis.createClient(REDIS_URL);
+const getAsync = promisify(client.get).bind(client);
 
 function getValueFromCache(key) {
-  const cachedRecord = cache[key];
-
-  if (cachedRecord && cachedRecord.expiryTimestamp) {
-    if (Date.now() <= cachedRecord.expiryTimestamp) {
-      log.info(`[cache] hit for ${key}`);
-      return cachedRecord.value;
-    }
-
-    log.info(`[cache] hit for ${key}, but was expired (${new Date(cachedRecord.expiryTimestamp)})`);
-    delete cache[key];
-    return null;
-  }
-
-  log.info(`[cache] miss for ${key}`);
-  return null;
+  return getAsync(key)
+    .then(function(result) {
+      return JSON.parse(result);
```

Comment:
I'd love to add some cache hit/miss logging here! It's helpful to debug and check up on when the app is running. Could have the same/similar format as the log lines from the app memory cache implementation.